habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/AbstractHabushuMojo.java

```diff
@@ -72,6 +72,13 @@ public abstract class AbstractHabushuMojo extends AbstractMojo {
     @Parameter(property = "habushu.testDirectory", required = true, defaultValue = "${project.basedir}/tests")
     protected File testDirectory;
 
+    /**
+     * Folder in which Python build files are located - should align with Poetry's
+     * project structure conventions.
+     */
+    @Parameter(property = "habushu.targetDirectory", required = true, defaultValue = "${project.basedir}/target")
+    protected File targetDirectory;
```

Comment:
A: this should no longer be needed - was from attempting automated testing for this feature